PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -438,28 +438,20 @@ private void WidthSlider_ValueChanged(object sender, RoutedPropertyChangedEventA
                 var timerBody = GetShapeByName(TimerLabConstants.TimerBodyId);
                 float increment = value - timerBody.Width;
 
-                if (increment == 0)
-                {
-                    ReformMarkersIfMissing();
-                }
-                else
-                {
-                    var lineMarkerGroup = GetLineMarkerGroup();
-                    var timeMarkerGroup = GetShapeByName(TimerLabConstants.TimerTimeMarkerGroupId);
-                    timerBody.Select();
-                    if (lineMarkerGroup != null)
```

Comment:
The check for line marker still needs to be there in case there is no line for the timer (i.e. less than 10 seconds)